packages/@lwc/engine/src/framework/attributes.ts

```diff
@@ -70,17 +67,12 @@ export const globalHTMLProperties: {
     },
     className: {
         attribute: 'class',
-        error: `Using the \`className\` property is an anti-pattern because of slow runtime behavior and potential conflicts with classes provided by the owner element. Use the \`classList\` API instead.`,
+        error:
+            'Using the `className` property is an anti-pattern because of slow runtime behavior and potential conflicts with classes provided by the owner element. Use the `classList` API instead.',
     },
     contentEditable: {
         attribute: 'contenteditable',
     },
-    isContentEditable: {
-        readOnly: true,
-    },
-    contextMenu: {
```

Comment:
Removed `contextMenu` because it's obsolete:
```
Elements in the following list are entirely obsolete, and must not be used by authors:
https://html.spec.whatwg.org/multipage/obsolete.html#attr-contextmenu
```